index.html

```diff
@@ -175,9 +172,10 @@ <h2 class="skills-pack-font">Skills</h2><br>
   </ul>
 </div>
 </section>
-<footer class="footer">
-  <h2 class="footer-headline">I'm always interested in hearing about new projects, 
-    so if you'd like to chat please get in touch.</h2>
+<footer class="footer">  
+     <div class="desktop-flex">
+       <h2 class="footer-headline">I'm always interested in hearing <br> about new projects, 
+    so if you'd <br> like to chat please get in touch.</h2>
```

Comment:
- [ ] On desktop versions between 768px and 1080, the contact section is skewed to the left because it doesn't take up the entire available width. Please fix it 🙏

### Your Design
![image](https://user-images.githubusercontent.com/46839548/166019378-374d5fef-b127-4ee3-8f75-9e819d6c1059.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166014267-3a2570fc-4880-474b-a1be-d0a35b8f19d2.png)

Reply:
fixed, thanks